explorer/src/Stories/AccordionMenu.elm

```diff
@@ -0,0 +1,74 @@
+module Stories.AccordionMenu exposing (..)
+
+import Config exposing (Config, Msg(..))
+import Css exposing (cursor, fontSize, listStyle, marginRight, marginTop, middle, none, paddingLeft, pointer, rem, verticalAlign, width)
+import Css.Global exposing (children, typeSelector)
+import Html.Styled exposing (div, li, span, text, ul)
+import Html.Styled.Attributes exposing (css)
+import Nordea.Components.AccordionMenu as AccordionMenu exposing (..)
+import Nordea.Resources.Icons as Icons
+import UIExplorer exposing (UI)
+import UIExplorer.Styled exposing (styledStoriesOf)
+
+
+stories : UI Config Msg {}
+stories =
+    styledStoriesOf
+        "AccordionMenu"
+        [ ( "Default"
+          , \_ ->
+                AccordionMenu.view
+                    [ Status Closed ]
+                    []
+                    [ AccordionMenu.header [] [ text "Kontrakt e-signert 10.10.22" ]
+                    , AccordionMenu.content []
+                        [ ul []
+                            [ li [] [ text "Undertegner identifisert 09.10.22" ]
+                            , li [] [ text "E-signering utloper snart 08.10.22" ]
+                            , li [] [ text "E-signering sendt 07.10.22" ]
+                            ]
+                        ]
+                    ]
+          , {}
```

Comment:
I have followed the structure you proposed but instead of a record for required arguments, I opted for optional arguments for everything including the initial state.